src/Umbraco.Web/UmbracoHelper.cs

```diff
@@ -146,7 +146,7 @@ public IHtmlString RenderTemplate(int contentId, int? altTemplateId = null)
         /// <returns></returns>
         public IHtmlString RenderMacro(string alias)
         {
-            return ComponentRenderer.RenderMacro(AssignedContentItem?.Id ?? 0, alias, new { });
+            return ComponentRenderer.RenderMacro(AssignedContentItem?.Id ?? 0, alias, new Dictionary<string, object>());
```

Comment:
Ideally we use a static reference to an `EmptyDictionary` reference so we aren't allocating a new instance everytime a macro is rendered, else we can pass in null and then deal with null in the `RenderMacro`. It's minor so i'll fix up.